doc/material-migration.md

```diff
@@ -14,20 +14,23 @@ We have standardized the naming of styles and resource keys. This change aims to
 
```

Comment:
Maybe this text can be adjusted a bit now I think @Xiaoy312 
![image](https://github.com/user-attachments/assets/19f31ea6-1e36-4c16-8fd5-e0ba59144fd9)

Reply:
i think it is fine.

Reply:
After another look, i am fine with it also now